tsdfileapi/api.py

```diff
@@ -975,7 +975,6 @@ def initialize(self, backend: str, namespace: str, endpoint: str) -> None:
         self.allow_info = self.backend_config["allow_info"]
         self.allow_delete = self.backend_config["allow_delete"]
         self.allow_rpc = self.backend_config.get("allow_rpc")
-        self.export_max = self.backend_config["export_max_num_list"]
```

Comment:
Should this option be removed from the various config files referencing and defaults.py too?

Reply:
will do